oak_functions/abi/src/lib.rs

```diff
@@ -73,6 +73,7 @@ pub struct ReportMetricRequest {
 #[cfg(feature = "oak-metrics")]
 pub enum ReportMetricError {
     ProxyAlreadyConsumed,
+    DeOrSerializingFailed,
```

Comment:
Makes sense, but I think the naming is a bit odd. I haven't looked around at how tarpc or grpc or capnproto or other RPC-like systems do, but I think it would be useful to do that to see if there is any best practice we can learn from them.